src/User/Console/SetPasswordCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/*
+ * This file is part of Flarum.
+ *
+ * For detailed copyright and license information, please view the
+ * LICENSE file that was distributed with this source code.
+ */
+
+namespace Flarum\User\Console;
+
+use Flarum\Console\AbstractCommand;
+use Flarum\Console\AskQuestionTrait;
+use Flarum\User\UserRepository;
+use Flarum\User\UserValidator;
+use Illuminate\Validation\ValidationException;
+
+class SetPasswordCommand extends AbstractCommand
+{
+    use AskQuestionTrait;
+
+    protected $userRepository;
+
+    protected $userValidator;
+
+    /**
+     * @param UserRepository $userRepository
+     */
+    public function __construct(UserRepository $userRepository, UserValidator $userValidator)
```

Comment:
If the class is type hinted already, there's no need to add a phpdoc. Because - as you can see - it makes code maintenance harder; you are missing one argument already 👍 

```suggestion
    public function __construct(UserRepository $userRepository, UserValidator $userValidator)
```